### DISCUSSION TITLE
Keine Verbindung nach Teile tauschen

### DISCUSSION THREAD

**Discussion by struppie87** [solectrus/solectrus#3436] [Q&A]
Guten Morgen,

Bei unserem Senec Speicher wurde gestern die Netzwerkkarte getauscht, jetzt habe ich das Problem das Solectrus immer schreibt keine Verbindung.
Dem Speicher habe ich die alte IP gegeben, der hatte eine andere nach dem Tausch.
Auch alle Komponenten (Router, Speicher und Raspberry Pi) haben schon einen Neustart durch.

Jetzt weiß ich nicht mehr weiter.

Danke

**Comment by ledermann** [comment]
Die neue Karte kommt mit einem alten Softwarestandand und läuft noch auf `http`. Du musst also den SENEC-Collector anpassen (`SENEC_SCHEMA=http`). Wir hatten das schon mal:

https://github.com/orgs/solectrus/discussions/2877
https://github.com/orgs/solectrus/discussions/3395
